src/gui/updownratiodlg.cpp

```diff
@@ -70,8 +70,8 @@ UpDownRatioDlg::UpDownRatioDlg(bool useDefault, qreal initialRatioValue,
     m_ui->timeSpinBox->setValue(initialTimeValue);
 
     connect(m_ui->buttonGroup, SIGNAL(buttonClicked(int)), SLOT(handleRatioTypeChanged()));
```

Comment:
and this?
`connect(m_ui->buttonGroup, static_cast<void (QButtonGroup::*)(int)>(&QButtonGroup::buttonClicked), this, &UpDownRatioDlg::handleRatioTypeChanged);`